streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamPartitionAssignor.java

```diff
@@ -408,14 +437,24 @@ public Subscription subscription(Set<String> topics) {
                 for (AssignedPartition partition : assignedPartitions) {
                     active.add(partition.taskId);
                     activePartitions.add(partition.partition);
+                    HostInfo hostInfo = consumerEndPointMap.get(processId);
```

Comment:
Just clarifying: if some instances specify the host info config while some others of the same application do not, this will just cause those who do not specify to not be queryable but not affecting those who do specify the configs right?